packages/babel-presets/babel-preset-kyt-core/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "no-var": 0
```

Comment:
FWIW these `.eslintrc.json`s aren't even honored right now since the root package's `lint` ignores them (to work around needing to get the eslint config cascade working)